static/machtsinn.js

```diff
@@ -161,3 +161,19 @@ var main = (function () {
 document.addEventListener('DOMContentLoaded', function () {
   main.init()
 })
+document.addEventListener("keypress", keyPressHandler, false);
+
+/* Button press handlers */
+function keyPressHandler(e){
+    if (e.keyCode == 78) {
```

Comment:
These (78/83/82) are the keycodes of capital 'N'/'R'/'S' while the tooltip says 'n'/'r'/'s' (which are simpler to type). Maybe allow both?

Reply:
I disagree - this are key codes for the 'keys' s/n/r. Capital letters are a combination of shift+key and are read accordingly

Reply:
According to this, that is the behaviour for keydown/keyup but not keypress:
https://stackoverflow.com/questions/11030532/keypress-and-keyup-why-is-the-keycode-different